libs/core/src/lib/dialog/dialog.component.scss

```diff
@@ -22,4 +22,9 @@
             border-top-right-radius: 0;
         }
     }
+
+    .fd-dialog__body--no-horizontal-padding {
```

Comment:
I think this should be in fundamental-styles, oddly it is mentioned in the documentation there but this class is missing

Reply:
https://github.com/SAP/fundamental-styles/pull/3957